satellitevu/apis/otm.py

```diff
@@ -1,5 +1,5 @@
 from datetime import datetime
-from typing import Literal, Optional, Tuple, Union
+from typing import Any, Literal, List, Optional, Tuple, Union
```

Comment:
can't do things like `Optional[list[str]]` so have to use `List`

Reply:
Could you do `list[str] | None` instead?

Reply:
oh yeah 🤦‍♀️

Reply:
i remember why- `list[str] | None` fails in 3.8